src/main/ks-installer/README.md

```diff
@@ -0,0 +1,105 @@
+# Ks-installer Helm Chart
```

Comment:
Replace the Chart with `KubeSphere` instead of `Ks-installer Helm Chart`. See an example of J[enkins Helm Chart](https://github.com/jenkinsci/helm-charts/tree/main/charts/jenkins).

Reply:
The modification has been completed.